validation-test/compiler_crashers_fixed/28617-child-source-range-not-contained-within-its-parent-guard-stmt.swift

```diff
@@ -5,6 +5,6 @@
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// RUN: not --crash %target-swift-frontend %s -emit-ir
+// RUN: not %target-swift-frontend %s -emit-ir
 {{guard let c:
 sil_scope
```

Comment:
This is accidental resolution.
There must be a range containment problem in the Parser.